src/dbnode/integration/admin_session_fetch_blocks_test.go

```diff
@@ -178,6 +179,9 @@ func testSetupToSeriesMaps(
 	for shardID, metadatas := range metadatasByShard {
 		blocksIter, err := session.FetchBlocksFromPeers(nsMetadata, shardID,
 			consistencyLevel, metadatas, resultOpts)
+		if err != nil {
+			panic(err)
```

Comment:
We perform a `require.NoError(t, err)` just below - whats the benefit of a panic here just before the check?

Reply:
Fixed